src/node.js

```diff
@@ -93,148 +107,108 @@ module.exports = class Node {
       this.env.IPFS_PATH = this.path
     }
 
-    run(this.exec, ['init', '-b', keySize], {env: this.env})
-      .on('error', done)
-      .pipe(bl((err, buf) => {
-        if (err) return done(err)
-
-        configureNode(this, this.opts, (err) => {
-          if (err) {
-            return done(err)
-          }
+    this._run(['init', '-b', keySize], {env: this.env}, (err, result) => {
+      if (err) {
+        return callback(err)
+      }
 
-          this.clean = false
-          this.initialized = true
+      configureNode(this, this.opts, (err) => {
+        if (err) {
+          return callback(err)
+        }
 
-          done(null, this)
-        })
-      }))
+        this.clean = false
+        this.initialized = true
+        callback(null, this)
+      })
+    })
 
     if (this.disposable) {
       shutdown.addHandler('disposable', 1, this.shutdown.bind(this))
     }
   }
 
   // cleanup tmp files
-  // TODO: this is a bad name for a function. a user may call this expecting
-  // something similar to "stopDaemon()". consider changing it. - @jbenet
-  shutdown (done) {
-    if (!this.clean && this.disposable) {
-      rimraf(this.path, (err) => {
-        if (err) throw err
-        done()
-      })
+  shutdown (callback) {
```

Comment:
There is a comment above which kinda describes what it does, no other docs currently, correct